src/main/kotlin/com/devfloor/untitled/option/application/OptionService.kt

```diff
@@ -0,0 +1,28 @@
+package com.devfloor.untitled.option.application
+
+import com.devfloor.untitled.option.domain.OptionRepository
+
+import com.devfloor.untitled.option.domain.Option
+import com.devfloor.untitled.option.domain.OptionType
+
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class OptionService(
+    private val repository: OptionRepository
+) {
+    @Transactional(readOnly = true)
+    fun show(options: List<String>): List<Option> {
+        val optionList = mutableListOf<Option>()
+        for(index in options.indices) {
+            optionList.add(repository.findByType(showOptionType(options[index])))
+        }
+        return optionList
+    }
```

Comment:
아래와 같이 변경하면 더 깔끔해질 것 같아요 !
```suggestion
    fun show(options: List<String>): List<Option> {
        return options.map {
            repository.findByType(showOptionType(it))
        }
    }
```